### PR TITLE
Translate kustomization.yaml

### PR DESCRIPTION
**Pull request by sunny0826**
@Liujingfang1 @monopole If the docs is updated, please contact me by email:<sunnydog0829@gmail.com> or issue：https://github.com/kubernetes-sigs/kustomize/issues/1001 or slack in *#kustomize*, I will update it in time.

### CODE REVIEW

**Comment by Liujingfang1** [comment]
@sunny0826 Thank you for updating the Chinese doc. Are you interested in adding Chinese version of examples? https://github.com/kubernetes-sigs/kustomize/tree/master/examples

**Comment by sunny0826** [comment]
@Liujingfang1 Of course, there are translation examples in my TODO list. In fact, I have applied kustomize to our company's CI/CD workflow and have written several blogs about it. I will translate the examples later.

File: docs/zh/kustomization.yaml [hunk 1]
```diff
@@ -0,0 +1,287 @@
...
+  app: bingo
+
+# 和 Labels 一样， 增加 Annotations
+# 为 key:value 键值对
+commonAnnotations:
+  oncallPager: 800-555-1212
+
+# 此列表中的每条记录都必须是一个存在的 YAML 资源描述文件
+# 这些 YAML 资源描述文件将被 kustomize 读取，修改和发布
+# 资源由文档标记（"---"）分隔
```
Comment (Liujingfang1) [inline docs/zh/kustomization.yaml]: I'm not quite familiar with the Chinese names for k8s resources.
By reading the paragraph, I suggest to change it to something like this for easy
readability.
```
# 此列表中的每条记录都必须是一个存在的 YAML 资源描述文件
# 一个YAML 资源描述文件可以含有多个由（“---”）分隔的资源。
# kustomize 将读取这些YAML文件中的资源，对其进行修改并
# 发布在kustomize的输出中。
```

File: docs/zh/kustomization.yaml [hunk 2]
```diff
@@ -0,0 +1,287 @@
...
+resources:
+- some-service.yaml
+- sub-dir/some-deployment.yaml
+
+# 列表中的每个条目都将创建一个 ConfigMap （它是n个映射的生成器）
```
Comment (Liujingfang1) [inline docs/zh/kustomization.yaml]:
```
# 列表中的每个条目都将创建一个 ConfigMap （它是n个 ConfigMap 的生成器）
```

File: docs/zh/kustomization.yaml [hunk 3]
```diff
@@ -0,0 +1,287 @@
...
+    - tls.crt=catsecret/tls.cert
+    - tls.key=secret/tls.key
+  type: "kubernetes.io/tls"
+- name: env_file_secret
+  # 文件路径以 k=v 键值对的形式，每行一个键值对
+  envs:
+    - env.txt
+  type: Opaque
+
+# generatorOptions 修改所有 ConfigMap 和 Secret 的行为
```
Comment (Liujingfang1) [inline docs/zh/kustomization.yaml]:
```
# # generatorOptions 修改所有 ConfigMapGenerator 和 SecretGenerator 的行为
```

File: docs/zh/kustomization.yaml [hunk 4]
```diff
@@ -0,0 +1,287 @@
...
+- github.com/Liujingfang1/mysql
+- github.com/Liujingfang1/kustomize/examples/helloWorld?ref=test-branch
+
+# 此列表中的每个条目都应可以解析为部分或完整的资源定义文件
+#
+# 这些（也可能是部分的）资源文件中的 name 必须与已经通过 `resources` 加载的 name 字段匹配
+# 或者通过 `bases` 中的 name 字段匹配
+# 这些条目将用于 _patch_（修改）已知资源
+#
+# 最好做一些小的 patches
```
Comment (Liujingfang1) [inline docs/zh/kustomization.yaml]:
```
推荐使用小的 patches
```

File: docs/zh/kustomization.yaml (hunk 2 above)
Comment (sunny0826) [reply docs/zh/kustomization.yaml]: # one ConfigMap resource (it's a generator of n maps).
The meaning of maps  is actually ConfigMap !
I have not tested this entry yet.
thx

File: docs/zh/kustomization.yaml (hunk 4 above)
Comment (sunny0826) [reply docs/zh/kustomization.yaml]: This translation is much better than mine!
Can you speak Chinese? I mean your name looks like a Chinese name.
I encountered some confusion when translating the [glossary](https://github.com/kubernetes-sigs/kustomize/blob/master/docs/glossary.md). The role of the glossary is to disambiguate. I don't want my translation to bring more ambiguity.

Comment (Liujingfang1) [reply docs/zh/kustomization.yaml]: @sunny0826 Yes, I can speak Chinese. 
If you get better translation, feel free to update the doc.

**Pull request merged**